test_suite/generic/test_generic.py

```diff
@@ -114,6 +114,25 @@ def test_cert_product_version_is_correct(self, host):
             assert f'Version: {product_version}' in cert_version, \
                 'Inconsistent version in pki certificate'
 
+    def test_inittab_and_systemd(self, host):
+        """
+        Check default runlevel or systemd target
+        """
+        kernel_release = host.check_output('uname -r')
+
+        with host.sudo():
+            if host.package('systemd').is_installed:
+                assert '/lib/systemd/system/multi-user.target' in \
+                       host.check_output('readlink -f /etc/systemd/system/default.target'), \
+                    'Unexpected systemd default target'
+            else:
+                assert 'id:3:initdefault' in host.check_output("grep '^id:' /etc/inittab"), \
+                    'Unexpected default inittab "id"'
+
+                if 'el5' in kernel_release:
```

Comment:
I thought we said we were not going to support rhel 6 or lower. I doesn´t hurt to have this though

Reply:
True. I replicated the same logic that is in the original test, but we can ask Frank if this is still really needed or not (I would suppose it's not). In the case RHEL 6.0 is not going to be supported anymore, we will have to update some of the tests already migrated (not a big deal IMO).